src/Fable.Cli/ProjectCracker.fs

```diff
@@ -287,12 +288,446 @@ let getCrackedProjectsFromMainFsproj (projFile: string) =
+        FSharpCompilerServiceChecker.getProjectOptionsFromScript additionalArgumentsBy checkerGetProjectOptionsFromScript file source targetFramework
+
+    static member Create(config: NetFWInfoConfig) =
+        NetFWInfo(config.MSBuildHost)
+        
+type FsxBinder (netFwInfo: NetFWInfo, checker: FSharp.Compiler.SourceCodeServices.FSharpChecker) =
+
+    member this.GetProjectOptionsFromScriptBy(tfm, file, source) = async {
+      let dummy : FSharpCompilerServiceChecker.CheckerGetProjectOptionsFromScript<FSharp.Compiler.SourceCodeServices.FSharpProjectOptions, _> =
+        fun (file, source, otherFlags, assumeDotNetFramework) ->
+          checker.GetProjectOptionsFromScript(file, source, otherFlags = otherFlags, assumeDotNetFramework = assumeDotNetFramework)
+
+      let! (rawOptions, mapper) =
+        netFwInfo.GetProjectOptionsFromScript(dummy, tfm, file, source)
+
+      let rawOptions = { rawOptions with OtherOptions = mapper rawOptions.OtherOptions }
+
+      return rawOptions
+    }
+// END TEMP
+    
+let getProjectOptionsFromScript scriptFile =
+    let (checker: FSharpChecker) = FSharpChecker.Create(keepAssemblyContents=true)
+    let msbuildLocator = MSBuildLocator()
+    let netFwInfo = NetFWInfoConfig.Default(msbuildLocator) |> NetFWInfo.Create
+
+    let tfm = netFwInfo.LatestVersion () // or specify a .NET version like `"4.6.1"`
+    let fsxBinder = FsxBinder(netFwInfo, checker)
+
+    let input = System.IO.File.ReadAllText(scriptFile)
+    
+    fsxBinder.GetProjectOptionsFromScriptBy(tfm, scriptFile, input)
+    |> Async.RunSynchronously
+    
+let getCrackedProjectFromScript scriptFile =
+        let fsprojOptions = getProjectOptionsFromScript scriptFile
+        let parsedOptions =
+            fsprojOptions.OtherOptions
+            |> List.ofArray
+            |> List.map (fun line ->
+                if line.StartsWith("-r:") then
+                    let dllPath = line.Substring(3)
+                    match tryGetFablePackage dllPath with
+                    | Some fablePackage -> ScriptOption.FablePackage fablePackage
+                    | None -> ScriptOption.Dll dllPath
+                else
+                    ScriptOption.Other line
+            )
+    
+        let dllReferences =
+            let netcoreappPath = Path.GetDirectoryName(typeof<int>.GetTypeInfo().Assembly.Location)
+            
+            parsedOptions
+            |> List.map (function | ScriptOption.Dll dll -> Some dll | _ -> None)
+            |> List.choose id
+            |> fun dllRx ->
+                Array.fold (fun dllRs rf ->
+                    if not (List.exists (fun r -> Path.GetFileNameWithoutExtension(r) = rf) dllRs) then
+                       let dllPath = Path.Combine(netcoreappPath, (sprintf "%s.dll" rf))
+                       dllPath::dllRs
+                    else
+                        dllRs
+                ) dllRx Fable.Standalone.Metadata.references_core
```

Comment:
If you're adding here the basic System assemblies necessary to compile Fable projects, what's the rest of the code doing, just getting custom `#r` references and file `#load`s?

Also, `Metadata.references_core` only includes assembly names not their location in the machine. How is this solved.